keras/backend/mxnet_backend.py

```diff
@@ -1168,6 +1168,9 @@ def gather(reference, indices):
     indices = mx.sym.Cast(indices.symbol, dtype=reference.dtype)
     return KerasSymbol(mx.sym.take(reference.symbol, indices))
 
+@keras_mxnet_symbol
+def embedding(data, weight, input_dim, output_dim):
```

Comment:
Add a check.
if data is KerasSymbol then data.symbol else data
if weight is KerasSymbol then weight.symbol else weight